community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -532,6 +532,6 @@ public static BoltConnector boltConnector( String key )
      */
     private static int dynamicRecordDataSizeForAligningWith( int recordSize )
     {
-        return recordSize - InternalRecordFormatSelector.select().dynamic().getRecordHeaderSize();
+        return recordSize - DynamicRecordFormat.RECORD_HEADER_SIZE;
```

Comment:
That's true for now it's ok and will work as it is, but for future formats that can give some problems. What do you think about moving `unsupported.dbms.block_size.* ` properties away from and do verification when we using them and already know the format?